File: erezervisi_api/eRezervisi/eRezervisi.Common.Dtos/UserSettings/UserSettingsGetDto.cs
namespace eRezervisi.Common.Dtos.UserSettings
{
    public class UserSettingsGetDto
    {
        public long UserId { get; set; }
        public bool ReceiveEmails { get; set; }
        public bool ReceiveNotifications { get; set; }
        public bool MarkObjectAsUncleanAfterReservation { get; set; }
    }
}

File: erezervisi_desktop/erezervisi_desktop/lib/models/responses/user/user_settings_get_dto.dart
class UserSettingsGetDto {
  late num userId;
  late bool receiveEmails;
  late bool receiveNotifications;
  late bool markObjectAsUncleanAfterReservation;

  UserSettingsGetDto({
    required this.userId,
    required this.receiveEmails,
    required this.receiveNotifications,
    required this.markObjectAsUncleanAfterReservation
  });

  factory UserSettingsGetDto.fromJson(Map<String, dynamic> json) {
    return UserSettingsGetDto(
        userId: json['userId'] as num,
        receiveEmails: json['receiveEmails'],
        receiveNotifications: json['receiveNotifications'],
        markObjectAsUncleanAfterReservation: json['markObjectAsUncleanAfterReservation'],);
  }
}
